source/features/scrollable-areas.css

```diff
@@ -2,7 +2,8 @@
 	.comment-body,
 	.markdown-body,
 	[data-testid='markdown-body'] {
-		:is(blockquote, pre) {
+		blockquote,
+		pre {
```

Comment:
Something cool about nested selectors is that they can replace `:is()` as well, sometime making it easier to read/format. 

In this case it was already nested, so `:is()` could just be dropped.